pkg/grpc/tracing/tracing.go

```diff
@@ -327,8 +327,8 @@ func GetProcessKprobe(event *MsgGenericKprobeUnix) *tetragon.ProcessKprobe {
 
 	proc, parent, tetragonProcess, tetragonParent := getProcessParent(&event.Msg.ProcessKey, event.Msg.Common.Flags)
 
-	// Set the ancestors only if --enable-process-kprobe-ancestors flag is set.
-	if option.Config.EnableProcessKprobeAncestors && proc.NeededAncestors() {
+	// Set the ancestors only if --enable-ancestors flag includes 'kprobe'.
+	if option.Config.EnableAncestors["kprobe"] && proc.NeededAncestors() {
```

Comment:
I think internally, it's fine to keep the many config static variables `EnableProcessKprobeAncestors` and co, so that it avoids these dynamic string to be used with a map. Because otherwise we should use constant string and that for access I'm not sure it would be optimal